packaging/linux/common/khiops_env/khiops_env.in

```diff
@@ -84,8 +84,15 @@ if command -v mpiexec &>/dev/null; then
     KHIOPS_MPI_ERROR=""
     KHIOPS_MPI_COMMAND="$(type -P mpiexec) $MPI_EXTRA_FLAGS @KHIOPS_MPI_EXTRA_FLAG@ @MPIEXEC_NUMPROC_FLAG@ $KHIOPS_PROC_NUMBER"
 else
-    KHIOPS_MPI_ERROR="We didn't find mpiexec in the regular path. Parallel computation is unavailable: Khiops is launched in serial"
-    KHIOPS_MPI_COMMAND=""
+    _MPIEXEC=@KHIOPS_PATH@mpiexec
```

Comment:
Comment here something along the lines of the commit message: That this is the fallback for conda, etc.

Reply:
Ok. Good point.

Reply:
Done. Note however that I have updated the commit message with a caveat.